src/lib.rs

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+pub use prost_types;
```

Comment:
```suggestion

pub use prost_types;
```

Please leave a blank line here to pass the license format check